README.md

```diff
@@ -9,6 +9,8 @@ Things you may want to cover:
 
 * System dependencies
 
+Use as username and password for the Database 'llopez'
```

Comment:
should be explained what db and that is the dev db to authorize via rake task. the easiest thing to do here is to change the name and password to the password we usually use for dev dbs check the repos.